CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+v0.7.0
+------
+* Update dependencies
```

Comment:
We should say we 

* Removed TileStache and Pillow dependencies
* Updated other dependency versions